app/models/certificate.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class Certificate < ApplicationRecord
-  belongs_to :attraction
+  has_and_belongs_to_many :attractions, optional: true
```

Comment:
ich bin absolut kein Freund von `has_and_belongs_to_many` und bin damit glaube auch nicht alleine:
https://stackoverflow.com/questions/6154462/habtm-best-practice
https://stackoverflow.com/questions/7850111/rails-3-1-has-and-belongs-to-many-deprecated

Reply:
@marcometz bitte schreib doch auch als Vorschlag noch, wie es besser wäre. Aus den Links würde es mir jetzt nicht eindeutig hervorgehen.

> HABTM is slightly out of date, and has been replaced with has_many :through
https://stackoverflow.com/a/6228768/9956365

Das?

Reply:
Ich habe es noch nie ausprobiert, und wollte mal schauen ob es fun ktioniert. Es funktioniert und scheint mir hier ausreichend zu sein.